### DISCUSSION TITLE
Tasmota + HC-SR04_DISTANCE + Relay

### DISCUSSION THREAD

**Discussion by amaljacob69** [arendst/Tasmota#19693] [Q&A]
I want to post this to ask for help. I'm creating a water tank monitoring system with Tasmota, using an HC-SR04 sensor and a relay. I need to activate the relay when the water level in the tank drops below 30cm and deactivate it when the water level reaches 80cm. Currently, I am able to retrieve readings from the sensor, and I can manually toggle the relay. How can I implement the automation described above? Thank you in advance.
<img width="290" alt="tasmota_configuration" src="https://github.com/arendst/Tasmota/assets/124233195/baaaaf35-ef83-47c6-ad94-270dd6424f29">
<img width="323" alt="tasmota_UI" src="https://github.com/arendst/Tasmota/assets/124233195/d4104beb-0ef0-4878-b061-0d0d5d2e9bfd">

**Comment by barbudor** [comment]
Rules
https://tasmota.github.io/docs/Rules/
